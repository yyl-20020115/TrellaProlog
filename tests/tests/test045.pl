:- initialization(main).

ok(N) :- write(ok), write(N).
ok(N) :- write(again), write(N).

main :- (true -> ok(1) ; write(nok1)), nl, fail.
main :- (false -> write(nok2) ; ok(2)), nl, fail.
main.
