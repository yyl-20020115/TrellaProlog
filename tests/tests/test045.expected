ok1
again1
ok2
again2
